.buildkite/scripts/steps/test/scout_test_run_builder.sh

```diff
@@ -8,7 +8,23 @@ source .buildkite/scripts/common/util.sh
 .buildkite/scripts/copy_es_snapshot_cache.sh
 
 echo '--- Discover Playwright Configs and upload to Buildkite artifacts'
-node scripts/scout discover-playwright-configs --save
+
+set +e;
+OUTPUT=$(node scripts/scout discover-playwright-configs --save --validate 2>&1)
+EXIT_CODE=$?
+set -e;
+
+if [[ $EXIT_CODE -ne 0 ]]; then
+  echo "Exiting with code 10 without retrying"
+  ERROR_MSG=$(echo "$OUTPUT" | grep -A 10 "ERROR" | sed -E 's/\x1b\[[0-9;]*m//g')
```

Comment:
maybe there is a nicer way of parsing the error from node.js script?

Reply:
Is there a need to hoist it up as an annotation?  I'm not opposed, but it adds complexity and we usually don't add static checks.

Reply:
I removed annotation and moved validation to Quick Checks (it was discussed with Tiago)